web/static/js/components/idea_controls.jsx

```diff
@@ -39,17 +39,21 @@ export const IdeaControls = props => {
     }
 
     if (currentUser.is_facilitator || currentUser.id === userId) {
+      const iconStyle = { color: idea.editing ? "lightgray" : "black" }
```

Comment:
ahhhh... we can actually simplify this nicely, as semantic allows "disabled" icons. disabled icons are grayed out AND they disable click events.

Reply:
peep https://semantic-ui.com/elements/icon.html#/definition

Reply:
as per our discussion, the built-in disabled feature only grays out the icon without disabling the onClick

Reply:
hey @tnewell5, rather than swapping out the color classes, we will still to conditionally apply 'disabled' class for consistency with semantic's styleguide.